### DISCUSSION TITLE
Build for aarch64

### DISCUSSION THREAD

**Discussion by ghost** [AsamK/signal-cli#1309] [General]
Would it be possible for someone to provide an aarch64 build? That would make it possible to use signal-cli with termux on mobile devices. What do you think about it?

**Comment by ghost** [comment]
Native libsignal builds for different architectures are linked here: 

https://github.com/AsamK/signal-cli/wiki/Provide-native-lib-for-libsignal

**Reply by m-ueberall** [reply]
Please note that you'd still need a JRE to use this; there are also native binaries available here (which include the above native library): https://github.com/AsamK/signal-cli/wiki/Binary-distributions#standalone-binaries-for-aarch64arm64-x86_64amd64-devices
